Package.swift

```diff
@@ -86,6 +94,11 @@ let package = Package(
                  * only.  Unconditionally define it instead.
                  */
                 .define("WIN32_LEAN_AND_MEAN"),
+                /*
+                 * These defines are required on WASM, to disable use of pthread.
```

Comment:
wording nit, also following capitalization defined in the WebAssembly spec
```suggestion
                 * These defines are required on Wasm/WASI, to disable use of pthread.
```